src/qt/sendcoinsdialog.cpp

```diff
@@ -306,17 +306,14 @@ bool SendCoinsDialog::PrepareSendText(QString& question_string, QString& informa
         formatted.append(recipientElement);
     }
 
-    if (model->wallet().privateKeysDisabled()) {
-        question_string.append(tr("Do you want to draft this transaction?"));
-    } else {
-        question_string.append(tr("Are you sure you want to send?"));
-    }
+    question_string.append(tr("Do you want to create this transaction?"));
 
     question_string.append("<br /><span style='font-size:10pt;'>");
-    if (model->wallet().privateKeysDisabled()) {
-        question_string.append(tr("Please, review your transaction proposal. This will produce a Partially Signed Bitcoin Transaction (PSBT) which you can save or copy and then sign with e.g. an offline %1 wallet, or a PSBT-compatible hardware wallet.").arg(PACKAGE_NAME));
-    } else {
-        question_string.append(tr("Please, review your transaction."));
+    if (model->getOptionsModel()->getMakeUnsignedButton() || model->wallet().privateKeysDisabled()) {
+        question_string.append(tr("Please review your transaction.<br/>The <b>Create Unsigned</b> button can be used to produce a Partially Signed Bitcoin Transaction (PSBT) which can be saved or copied and then signed with a PSBT-compatible wallet (e.g. an offline %1 wallet, or a PSBT-compatible hardware wallet.)").arg(PACKAGE_NAME));
+    }
+    if (!model->wallet().privateKeysDisabled()) {
+        question_string.append(tr("<br/>The <b>Send</b> button will create, sign, and broadcast this transaction."));
```

Comment:
The new prompt sounds too complicated for normal users...

Reply:
I've reverted it back to the original prompt with a slight modification